eosetl/mappers/join_split_mapper.py

```diff
@@ -0,0 +1,45 @@
+# MIT License
```

Comment:
Remove unused files and code. @Vasiliy-Bondarenko

Reply:
mostly done. i left the code that will be needed in the future. like streaming and tests.